easy_switch_user/controllers/__init__.py

```diff
@@ -0,0 +1 @@
+from . import main
```

Comment:
Don't remove copiryght, as this module is not ours, it's original author's decision